lib/verify.js

```diff
@@ -49,8 +49,19 @@ module.exports = {
       throw new TypeError('parsedSignature: unsupported algorithm ' +
                           parsedSignature.algorithm);
 
-    var hmac = crypto.createHmac(alg[1].toUpperCase(), secret);
+    var algUpper = alg[1].toUpperCase();
+    var hmac = crypto.createHmac(algUpper, secret);
     hmac.update(parsedSignature.signingString);
-    return (hmac.digest('base64') === parsedSignature.params.signature);
+    
+    // Compare hmac.digest('base64') with parsedSignature.params.signature in
+    // constant time. Double hmac verification is the preferred way to do this
+    // since we can't predict optimizations performed by the runtime.
+    // More information can be found here: http://tinyurl.com/doublehmac
```

Comment:
Sure, so could a regular URL. :) 

I'll change it back, but it was a long URL and wrapped beyond the width of the rest of the file. I was just trying to stick with the style.